packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -1156,39 +1155,31 @@ function invertBuilds(
  * @param fieldKinds - The field kinds to delegate to.
  */
 export function* relevantRemovedRoots(
-	{ change, revision }: TaggedChange<ModularChangeset>,
+	change: ModularChangeset,
 	fieldKinds: ReadonlyMap<FieldKindIdentifier, FieldKindWithEditor>,
 ): Iterable<DeltaDetachedNodeId> {
-	yield* relevantRemovedRootsFromFields(
-		change.fieldChanges,
-		revision,
-		change.nodeChanges,
-		fieldKinds,
-	);
+	yield* relevantRemovedRootsFromFields(change.fieldChanges, change.nodeChanges, fieldKinds);
 }
 
 function* relevantRemovedRootsFromFields(
 	change: FieldChangeMap,
-	revision: RevisionTag | undefined,
 	nodeChanges: ChangeAtomIdMap<NodeChangeset>,
 	fieldKinds: ReadonlyMap<FieldKindIdentifier, FieldKindWithEditor>,
 ): Iterable<DeltaDetachedNodeId> {
 	for (const [_, fieldChange] of change) {
-		const fieldRevision = fieldChange.revision ?? revision;
 		const handler = getChangeHandler(fieldKinds, fieldChange.fieldKind);
 		const delegate = function* (node: NodeId): Iterable<DeltaDetachedNodeId> {
 			const nodeChangeset = tryGetFromNestedMap(nodeChanges, node.revision, node.localId);
 			assert(nodeChangeset !== undefined, 0x931 /* Unknown node ID */);
 			if (nodeChangeset.fieldChanges !== undefined) {
 				yield* relevantRemovedRootsFromFields(
 					nodeChangeset.fieldChanges,
-					fieldRevision,
 					nodeChanges,
 					fieldKinds,
 				);
 			}
 		};
-		yield* handler.relevantRemovedRoots(tagChange(fieldChange.change, fieldRevision), delegate);
+		yield* handler.relevantRemovedRoots(fieldChange.change, delegate);
```

Comment:
Was this wrong before this change?

Reply:
Yes, it just wasn't being used. I've been trying to commit another PR that just ran into this. :)